sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 ### Breaking changes
 - Removed `GraphemeLength` and `GraphemeOffset` from `CategorizedEntity`, `SentenceSentiment`, and `LinkedEntityMatch`.
 - `GraphemeCount` in `TextDocumentStatistics` has been renamed to `CharacterCount`.
+- In both `DetectedLanguage` property `SentimentScores` has been renamed to `ConfidenceScores`.
+
+### Added
+- Added `Text` property to `SentenceSentiment`.
+- `Warnings` property exist on each document-level response object returned from the endpoints. It is a list of `TextAnalyticsWarnings`.
```

Comment:
nit: property exist => property exists

Reply:
Maybe rephrasing it to
`Warnings` properties added to each document-level response...